src/bundle/ui-dev/src/modules/sub-items/sub.items.module.js

```diff
@@ -867,7 +867,6 @@ export default class SubItemsModule extends Component {
     }
 
     renderDeleteConfirmationPopupFooter(selectionInfo) {
```

Comment:
I would rename all these methods/variables you modified, as you no longer render these elements - it's more like `getDeleteConfirmationBtns`